src/client/app/actions/version.ts

```diff
@@ -27,7 +27,7 @@ export function receiveVersion(data: string): t.ReceiveVersion {
  * @returns Whether preferences are fetching
```

Comment:
I think this comment should be update because it also checks if it already exists in state.